src/GenAPI/Microsoft.DotNet.GenAPI.Tool/Program.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using Microsoft.DotNet.GenAPI.Shared;
+
+
+namespace Microsoft.DotNet.GenAPI.Tool
+{
+    /// <summary>
+    /// CLI frontend for the Roslyn-based GenAPI.
+    /// </summary>
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            // Global options
+            Option<string> assemblyOption = new("--assembly",
+                "Delimited (',' or ';') set of paths for assemblies or directories to get all assemblies.")
+            {
+                ArgumentHelpName = "file,dir,...",
+                IsRequired = true
+            };
```

Comment:
System.CommandLine should already take care of parsing the input (which includes splitting multiple values that are separated by a delimiter). Can you please follow what we do in APICompat: https://github.com/dotnet/sdk/blob/4bd9b67d81fbf6d65012ef08c9d8cf2a27953057/src/ApiCompat/Microsoft.DotNet.ApiCompat.Tool/Program.cs#L47-L49